app/general/tests/test_views_institution.py

```diff
@@ -31,10 +39,12 @@ def test_institutions_view_correct_projects_returned(self):
         response = self.client.get(self.url)
 
         institutions = response.context["institutions"]
+
+        self.assertEqual(response.status_code, 200)
         self.assertEqual(len(institutions), 2)
-        self.assertTrue(all(hasattr(inst, "project_count") for inst in institutions))
-        self.assertEqual(institutions[0].project_count, 0)
-        self.assertEqual(institutions[1].project_count, 2)
+        self.assertTrue(all("project_count" in inst for inst in institutions))
+        self.assertEqual(institutions[0]["project_count"], 0)
+        self.assertEqual(institutions[1]["project_count"], 2)
```

Comment:
Please verify here the rating is correct.

Reply:
I'll add a separate test for rating